packages/framework-provider-kubernetes/src/index.ts

```diff
@@ -29,13 +28,22 @@ export const Provider: ProviderLibrary = {
   // ProviderAuthLibrary
   auth: {
     rawToEnvelope: undefined as any,
+    fromAuthToken: undefined as any,
   },
   // ProviderAPIHandling
   api: {
     requestSucceeded: undefined as any,
     requestFailed: undefined as any,
   },
+  connections: {
+    storeData: notImplemented as any,
+    fetchData: notImplemented as any,
+    deleteData: notImplemented as any,
+    sendMessage: notImplemented as any,
```

Comment:
I think it's better to leave as undefined

Reply:
Check https://github.com/boostercloud/booster/pull/373#discussion_r473965987